### ISSUE TITLE
Consolidate topics

### ISSUE THREAD

**Issue by davisdude** [davisdude/vim-love-docs#3]
Put enums and types under modules.

i.e.

```
1. love                             |love-love|
  1.1. graphics                     |love.graphics|
    1.1.1. modules                  |love.graphics-modules|
      1.1.1.1. draw                 |love.graphics.draw|
    1.1.2. enums                    |love.graphics-enums|
      1.1.2.1. fillMode             |love.graphics-fillMode| |love-fillMode|
    1.1.3. types                    |love.graphics-types|
      1.1.3.1. Canvas               |love.graphics-Canvas| |love-Canvas|
```

To do this, indent size would have to be decreased in order to not cut off names

Pros: 
- More organised
- More logical

Cons:
- More work
- Less clean (for table of contents, bc of spacing)

**Comment by davisdude** [comment]
Doing this also solves #1

**Comment by davisdude** [comment]
Closed with [a65102a98b7b8347a98b8](https://github.com/davisdude/vim-love-docs/commit/3bd6821d5b2e3efa3a7a65102a98b7b8347a98b8)
